<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Metodos</class>
 <widget class="QMainWindow" name="Metodos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Métodos de integración</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="temasBtn">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>300</y> 
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #009EBE;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color: #000000;
}</string>
    </property>
    <property name="text">
     <string>Ver Métodos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>10</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold Condensed</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Solución de métodos de integración.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>401</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    
	color: #1452AD;
}</string>
    </property>
    <property name="text">
     <string>Tema 4 Solución de métodos de integración
Consiste en elaborar un programa que dé solución a los 3 métodos vistos en este tema

    - Integración Trapezoidal

    - Integración Simpson 3/8
    
    - Integración Simpson 1/3 
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>470</y>
      <width>800</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    
	color: #056757;
}</string>
    </property>
    <property name="text">
     <string> -----------------------------------   UNAM, FES ARAGÓN, METODOS NUMERICOS 1359.   ----------------------------------- </string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
